Automattic/wp-calypso
# AL/Rewind: Clone Site interface

Alternate Restores for Rewind! Feature flagged to be available in all environments _except_ production for testing.

**Testing Instructions**

1. You are welcome to spin up a local development environment on this branch, or click the calypso.live link below.
2. Select a site which is rewind enabled with an active state (rewind buttons visible in the activity log).
3. Navigate to Settings > General and scroll to the bottom. Under Site Tools, you should have a "Clone" option. Click that button to enter the flow.
4. In step 2, provide your new site's title and public URL. Your public URL _must_ be a working WordPress install, and must be prefixed with `http://` or `https://`.
5. In step 3, provide the server credentials which correspond to the public URL entered in step 2. They must be active/working credentials. Make sure to add the server path that points to the install in the advanced settings.
6. Continue through the flow and ensure the restore runs and completes, either successfully or not. Restore failures are not a failure of this PR, but please do report those as separate issues in the `jetpack-backups` repository if necessary.
